os/drivers/ai-soc/ndp120/src/ndp120_api.c

```diff
@@ -937,6 +937,8 @@ void ndp120_show_debug(int include_spi)
 	}
 }
 
+int ndp120_init(struct ndp120_dev_s *dev, bool reinit);
```

Comment:
shouldn't this be at declaration section at the top?

Reply:
Done